modules/microshift-enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-workload-partitioning.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-enabling-workload-partitioning_{context}"]
+= Enabling workload partitioning
+
+To enable workload partiotioning on {microshift-short},make the following configuration changes:
+
+* Update and embed the Kubelet configuration file in the {microshift-short} `config.yaml` file to enable and configure CPU Manager for the workloads.
+* Update the CRI-O configuration file with the cpuset and management annotation.
+* Update the systemd configuration file with CPUAffinity setting for the {microshift-short} and CRI-O services.
+
+.Procedure
+
+. Update and embed the Kubelet configuration in the {microshift-short} `config.yaml` file:
+* Update the Kubelet configuration file in the path `/etc/kubernetes/openshift-workload-pinning` with the `cpuset` value. The Kubelet configuration instructs the kubelet to modify the node resources with the capacity and allocatable CPUs.
++
+.Example of Kubelet configuration
+[source,yaml]
+----
+# ...
+{
+  "management": {
+    "cpuset": "0,6,7"
+  }
+}
+# ...
+----
+* Update the {microshift-short} config.yaml file in the path `/etc/microshift/config.yaml`.Embed the Kubelet configuration in the {microshift-short} `config.yaml` file to enable and configure CPU Manager for the workloads.
++
+.Example of {microshift-short} `config.yaml` file
+[source,yaml]
+----
+# ...
+kubelet:
+  reservedSystemCPUs: 0,6,7<1>
+  cpuManagerPolicy: static
+  cpuManagerPolicyOptions:
+    full-pcpus-only: "true"<2>
+  cpuManagerReconcilePeriod: 5s
+# ...
+----
+<1> exclusive cpuset for the system daemons and the interrupts/timers.
+<2> kubelet configuration sets the `CPUManager policy` option to `full-pcpus-only` to ensure allocation of whole cores to a containers workload.
+
+. CRI-O configuration:
+* Update the CRI-O configuration file.
```

Comment:
its missing  the CRIO configuration file path, it should override the default configuration that already exists  by default `11-microshift-ovn.conf` so the config can be for example:  `/etc/crio/crio.conf.d/20-microshift-workload-partition.conf`  .

Reply:
I have made changes. let me know if the changes are correct.

I have few questions:
1. Is there any links or references to be provided for more information . eg:  building blueprint?
2. Should the example blue print be added? examples are already provided in sections in the current doc.

Reply:
yes the changes with the crio configuration path looks correct, 
1. most of the example for blueprints are using `[[customizations.files]]` which can be found [here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-command-line-interface_composing-a-customized-rhel-system-image?extIdCarryOver=true&sc_cid=701f2000001Css5AAC#specifying_customized_files_in_the_blueprint),
2. I dont think you need to provide the sample example separately in blueprint format.